[2023-12-10 19:35:50,070] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2022-12-17T00:00:00+00:00 [queued]>
[2023-12-10 19:35:50,116] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2022-12-17T00:00:00+00:00 [queued]>
[2023-12-10 19:35:50,119] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:35:50,120] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:35:50,121] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:35:50,277] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): tercera_tarea> on 2022-12-17 00:00:00+00:00
[2023-12-10 19:35:50,379] {standard_task_runner.py:52} INFO - Started process 1599 to run task
[2023-12-10 19:35:50,406] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'tercera_tarea', 'scheduled__2022-12-17T00:00:00+00:00', '--job-id', '421', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpmbq0efjt', '--error-file', '/tmp/tmp10p3llz0']
[2023-12-10 19:35:50,417] {standard_task_runner.py:80} INFO - Job 421: Subtask tercera_tarea
[2023-12-10 19:35:51,480] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2022-12-17T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:35:51,893] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=tercera_tarea
AIRFLOW_CTX_EXECUTION_DATE=2022-12-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-17T00:00:00+00:00
[2023-12-10 19:35:51,898] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:35:51,902] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 3 y sere corrida luego de Tarea 1 al mismo tiempo que Tarea 2']
[2023-12-10 19:35:51,922] {subprocess.py:85} INFO - Output:
[2023-12-10 19:35:51,926] {subprocess.py:92} INFO - hola, soy la tarea 3 y sere corrida luego de Tarea 1 al mismo tiempo que Tarea 2
[2023-12-10 19:35:51,929] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:35:52,085] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=tercera_tarea, execution_date=20221217T000000, start_date=20231210T193550, end_date=20231210T193552
[2023-12-10 19:35:52,192] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-10 19:35:52,286] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
